cobalt/browser/BUILD.gn

```diff
@@ -136,3 +138,29 @@ source_set("metrics") {
     "//url",
   ]
 }
+
+source_set("global_features") {
```

Comment:
BTW - you'll want to sync for this. I just submitted: https://github.com/youtube/cobalt/commit/47bc0edf69062c3ebb2989460175736569da04e9